src/main/resources/de/qabel/desktop/ui/about/about.css

```diff
@@ -1,21 +1,123 @@
-.button-about {
-    -fx-font-weight: bolder;
-    -fx-pref-width:220px;
+
+.btn-about {
+    -fx-border-color:none;
+    -fx-pref-width:500px;
+    -fx-pref-height:60px;
     -fx-border-width: 1px 1px 1px 1px;
-    -fx-text-alignment:left;
     -fx-background-repeat: no-repeat;
     -fx-background-image: url('/img/chevron_right.png');
     -fx-background-position:right;
-    -fx-font-size: 14px;
+    -fx-alignment: TOP_LEFT;
+    -fx-padding: 15px 100px 5px 10px;
+}
+
+.hyperlink-box-about {
+    -fx-alignment: BOTTOM_LEFT;
+    -fx-border-color: transparent;
+    -fx-padding: 0;
+    -fx-underline: false;
+}
+
+.hyperlink-box-about .text {
+    -fx-fill: #222222;
+    -fx-font-size: 11;
+}
+
+.hyperlink-box-about:visited .text {
```

Comment:
in normal css, you can concat all these pseudo-classes to a single statement like `.hyperlink-box-about:visited:hover:pressed .text {...}` or at least concat the classes to one statement like `.hyperlink-box-about .text, .hyperlink-box-about:visited .text, .hyperlink-box-about:hover .text, .hyperlink-box-about:pressed .text {...}`. This should be possible in javaFX, too